wazo_calld/plugins/calls/bus_consume.py

```diff
@@ -210,13 +210,21 @@ def _relay_channel_entered_bridge(self, event):
                 logger.debug('channel %s not found', participant_channel_id)
                 return
 
+            set_channel_id_var_sync(
+                self.ari,
+                participant_channel_id,
+                'WAZO_CONVERSATION_DIRECTION',
+                '',  # NOTE(afournier): it will be recalculated in `make_call_from_channel`
+                bypass_stasis=True,
+            )
```

Comment:
Maybe extract this to a function `invalidate_conversation_direction_cache` or equivalent?